ngraph_onnx/onnx_importer/utils/conv.py

```diff
@@ -51,7 +51,8 @@ def get_pads(onnx_node: 'NodeWrapper') -> Tuple[List[int], List[int]]:
     """
     auto_pad = onnx_node.get_attribute_value('auto_pad')
     pads = onnx_node.get_attribute_value('pads', ())  # Padding along each axis
-    kernel_shape = get_kernel_shape(onnx_node)
+    if(kernel_shape is None):
```

Comment:
Why `kernel_shape` may be None if you always pass it some value in `make_pooling_op`? The same for `get_strides`. Maybe checking whether `kernel_shape is None` should be done before passing its value to these functions.